app/views/_container-statuses.html

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
Curious about these comments here, these variables are not in the directive scope?